soe/src/stanford.make

```diff
@@ -152,7 +152,7 @@ projects[stanford_manage_content][download][url] = "git@github.com:SU-SWS/stanfo
 projects[stanford_manage_content][subdir] = "stanford"
 projects[stanford_manage_content][type] = "module"
 
-projects[stanford_news][download][tag] = "7.x-3.6+23-dev"
+projects[stanford_news][download][tag] = "7.3.7-alpha.5"
```

Comment:
```suggestion
projects[stanford_news][download][tag] = "7.x-3.7-alpha5"
```

Reply:
Pretty sure drush make needs the drupal syntax of tags.

Reply:
@mdyoung3 what about this one? I might be wrong about `drush make` needing the Drupal tag syntax, but we probably should stick to it rather than mixing Drupal and semver.

Reply:
@jbickar ,

I'll defer to you on this. 

Pushing a fixup!

Reply:
I believe drush make needs to reference the git tag name and not the module version for this work.

Reply:
Hi @sherakama Those are the tag names. 

![Screen Shot 2020-11-19 at 9 55 43 AM](https://user-images.githubusercontent.com/8387457/99697910-675e1700-2a4d-11eb-8373-94dbafaf0ef4.png)